mtools/mloginfo/sections/connection_section.py

```diff
@@ -84,5 +148,16 @@ def run(self):
             opened = ip_opened[ip] if ip in ip_opened else 0
             closed = ip_closed[ip] if ip in ip_closed else 0
 
-            print "%-15s  opened: %-8i  closed: %-8i" % (ip, opened, closed)
+            if genstats:
+                covered_count = ipwise_count[ip] if ip in ipwise_count else 1
+                connection_duration_ip = ipwise_sum_durations[ip] if ip in ipwise_sum_durations else 0
+
+                print "%-15s  opened: %-8i  closed: %-8i dur-avg(s): %-8i dur-min(s): %-8i dur-max(s): %-8i" % (ip, opened, closed, connection_duration_ip/covered_count, ipwise_min_connection_duration[ip], ipwise_max_connection_duration[ip])
+            else:
+                print "%-15s  opened: %-8i  closed: %-8i" % (ip, opened, closed)
+
         print
+
+        if genstats and fullconn_counts > 0:
+            print "Average connection duration across all IPs %d seconds, Minimum duration %d , Maximum duration %d" % (sum_durations/fullconn_counts, min_connection_duration, max_connection_duration)
```

Comment:
To match the output style of other `mloginfo` sections, I would lowercase Minimum and Maximum.

I think the order of output here should also match the info above (avg/min/max) and be in a similar format for consistency.

Reply:
done